pkg/catalog/catalog.go

```diff
@@ -6,18 +6,22 @@ import (
 	"github.com/weaveworks/profiles/api/v1alpha1"
 )
 
+// Catalog contains profiles which were gathered from the underlying cluster.
```

Comment:
I think this might be better represented as...

```Catalog provides an in-memory cache of profiles from the cluster which can be queried easily."

What do you think?

Reply:
I think you are missing something from this comment. :) @bigkevmcd

Reply:
Hah...a typo in my markup meant that GitHub didn't render it :-)

Reply:
yeah, that sounds good. :)